src/NuGet.Core/NuGet.Common/XmlUtility.cs

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Xml;
-using System.Xml.Linq;
-
-namespace NuGet.Common
-{
-    public static class XmlUtility
```

Comment:
Just pointing out that this is an API breaking change.

Personally I don't care because NuGet is not an XML processing library, so I don't think customers should be depending on us to provide this functionality. If someone disagrees, then we'll need to bring this class back and make it a wrapper for the internal class.

Reply:
I'm more concerned about interplay between different versions of the NuGet assemblies.

There's been instances like because of sleet. Not sure how big of a concern should that be.